atomate/vasp/workflows/base/lobster.py

```diff
@@ -0,0 +1,219 @@
+        additional_outputs (list): list of additional files to be stored in the
+            results DB. They will be stored as files in gridfs. Examples are:
+            "ICOHPLIST.lobster" or "DOSCAR.lobster". Note that the file name
+            should be given with the full name and the correct capitalization.
+
+    Returns: Workflow
+
+
+    """
+
+    c = c or {}
+
+    vasp_cmd = c.get("VASP_CMD", VASP_CMD)
+    lobster_cmd = c.get("LOBSTER_CMD", LOBSTER_CMD)
+    db_file = c.get("DB_FILE", DB_FILE)
+
+    fws = []
+    staticfw = StaticFW(structure=structure,
+                        vasp_input_set=LobsterSet(structure, user_incar_settings=user_incar_settings,
+                                                  user_kpoints_settings=user_kpoints_settings, isym=isym),
+                        vasp_cmd=vasp_cmd, db_file=db_file)
+    fws.append(staticfw)
+    fws.append(LobsterFW(structure=structure, parents=staticfw, calculationtype=calculationtype,
+                         delete_wavecar=delete_all_wavecars,
+                         delete_wavecar_previous_fw=delete_all_wavecars, lobster_cmd=lobster_cmd,
+                         db_file=db_file, lobsterin_key_dict=user_lobsterin_settings,
+                         user_supplied_basis=user_supplied_basis, handler_group="default",
+                         validator_group="strict",
+                         additional_outputs=additional_outputs))
+
+    workflow = Workflow(fws, name="LobsterWorkflow")
+    return workflow
+
+
+# TODO: include address_min_basis and address_max_basis correctly
+# extend test
+def get_wf_lobster_test_basis(structure: Structure, calculationtype: str = 'standard', delete_all_wavecars: bool = True,
+                              c: dict = None,
+                              address_max_basis: str = os.path.join(MODULE_DIR, "basis_lobster/PBE_54_max_basis.yaml"),
+                              address_min_basis: str = os.path.join(MODULE_DIR, "basis_lobster/PBE_54_min_basis.yaml"),
+                              user_lobsterin_settings: dict = None,
+                              user_incar_settings: dict = None,
+                              user_kpoints_settings: dict = None,
+                              isym: int = -1, additional_outputs: List[str] = None) -> Workflow:
+    """
+    creates workflow where all possible basis functions for one compound are tested
+    at the end, the user has to decide which projection worked best (e.g., based on chargespilling)
+    this is the recommended workflow at the moment!
+    Args:
+        structure (Structure): structure object that will be used during the run
+        calculationtype (str): only "standard" is implemented so far
+        delete_all_wavecars (bool): all wavecars wil be delted if True
+        c (dict): specifications for wf, e.g. VASP_CMD, LOBSTER_CMD etc.
+        address_max_basis (str): address to yaml file including maximum basis set

```

Comment:
Why is this an argument? I'm just trying to understand the rationale of this in the code.

If there are multiple min and max basis sets per potcar version, then I think this would be better as just a key for a dictionary. I.e., `PBE_54`. The dictionary specifying all the basis sets could be loaded in the init file of a module.

See my note below about moving the basis set information to pymatgen. I think then atomate should just select the specific min and max basis using a simple key.

If this doesn't make sense then maybe we could have a zoom meeting.

Reply:
Will change this but will keep the addresses for future changes.